src/core/annotation.js

```diff
@@ -439,13 +439,40 @@ class Annotation {
     );
   }
 
-  isHidden(annotationStorage) {
+  /**
+   * Check if the annotation must be printed in taking into account
```

Comment:
Nit: Here, and in the comment below, let's replace the word `in` with `by` to make this sentence flow a little easier:
```javascript
/**
 * Check if the annotation must be printed by taking into account the value
 * found in the annotationStorage which may have been set through JS.
 * 
```